silx/gui/plot/items/histogram.py

```diff
@@ -297,6 +301,20 @@ def setData(self, histogram, edges, align='center', copy=True):
 
         self._updated(ItemChangedType.DATA)
 
+    def setBaseline(self, baseline):
```

Comment:
Same as for `Curve`, this should be merged with set data, or you have to deal with curve and baseline of different length when updating...